website/static/css/screen/05-mixins.scss

```diff
@@ -9,3 +9,217 @@
+ * layout, but with increased font size for paragraph text.
+ */
+@mixin person {
+    box-sizing: border-box;
+    width: 100%;
+    margin: 0 auto 30px;
+
+    @media (min-width: 960px) {
+        width: percentage(1/3);
+        padding: 10px;
+        display: inline-block;
+        vertical-align: top;
+    }
+
+    &:target {
+        background: #333;
+        border-radius: 5px;
+        padding: 9px;
+    }
+
+    > h3 {
+        margin-bottom:          25px;
+        text-align:             center;
+        > img {
+            border:                 10px solid #fff;
+            display:                block;
+            margin:                 0  auto 20px;
+        }
+        > em {
+            font-weight:            bold;
+            display:                block;
+        }
+    }
+
+    > .stats {
+        margin:                 40px 0;
+        @include                easy-contain;
+
+        > li:first-child {
+            border-bottom:          1px solid #fff;
+            font-size:              32px;
+            font-weight:            bold;
+            margin-bottom:          5px;
+            padding-bottom:         3px;
+        }
+        > .secondary {
+            display:                table-cell;
+            float:                  left;
+            font-size:              13px;
+            padding:                0 10px 0 70px;
+            width:                  50px;
+            vertical-align:         middle;
+            > em {
+                color:                  #e5352d;
+                float:                  left;
+                font:                   50px
+                                        'League Gothic',
+                                        'Helvetica Neue Condensed',
+                                        'Arial',
+                                        sans-serif;
+                margin-left:            -60px;
+                text-align:             center;
+                width:                  60px;
+                vertical-align:         middle;
+            }
+        }
+        > .alt {
+            border-left:            1px solid #fff;
+            margin-left:            9px;
+        }
+    }
+
+    /* Biography, quotes and further reading URL. */
+    p {
+        font-size:              16px;
+        line-height:            24px;
+        margin-bottom:          16px;
+        @media (min-width: 960px) {
+            font-size: 13px;
+            line-height: 18px;
+        }
+    }
+
+    > .quote {
+        margin:                40px 0 0;
+        text-align:            center;
+
+        blockquote {
+            font:                   18px Georgia, serif;
+            line-height:            28px;
+            margin-bottom:          20px;
+            padding:                0 40px;
+            position:               relative;
+        }
+        blockquote:before,
+        blockquote:after {
```

Comment:
Since we're using nesting a lot here, how about nesting these rules too? e.g. `blockquote { &:before { … } }`

Also, since these are pseudo-elements instead of pseudo-classes, should we be using `::before` instead of `:before`, etc.? Looking at Google Analytics and the [MDN browser support data][1], I can only find 4 visitors in the last month (out of 5,636) whose browser doesn't support the new syntax (1 visitor on IE 7, 3 on IE 8).

[1]: https://developer.mozilla.org/en/docs/Web/CSS/Pseudo-elements#Notes